tests/test_init.py

```diff
@@ -16,40 +16,33 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 import unittest
 
+import mock
+
 import skt
 
 
+def mocked_requests_get(*args):
+    """Function to handle mocked HTTP requests"""
+    class MockResponse(object):
+        # pylint: disable=too-few-public-methods
+        """MockResponse class for returning HTTP requests"""
+        def __init__(self, text_data, status_code):
+            self.text = text_data
+            self.status_code = status_code
+
+    if args[0] == 'https://patchwork.kernel.org/patch/10326957/mbox/':
+        return MockResponse("mbox text goes here", 200)
+
+    return MockResponse(None, 404)
+
+
 class TestInit(unittest.TestCase):
     """Test cases for skt's __init__.py"""
 
-    def test_stringify_with_integer(self):
-        """Ensure stringify() can handle an integer"""
-        myinteger = int(42)
-        result = skt.stringify(myinteger)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, str(myinteger))
-
-    def test_stringify_with_string(self):
-        """Ensure stringify() can handle a plain string"""
-        mystring = "Test text"
-        result = skt.stringify(mystring)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, mystring)
-
-    def test_stringify_with_unicode(self):
-        """Ensure stringify() can handle a unicode byte string"""
-        myunicode = unicode("Test text")
-        result = skt.stringify(myunicode)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, myunicode.encode('utf-8'))
-
-    def test_parse_bad_patchwork_url(self):
-        """Ensure parse_patchwork_url() handles a parsing exception"""
-        patchwork_url = "garbage"
-        with self.assertRaises(Exception) as context:
-            skt.parse_patchwork_url(patchwork_url)
-
-        self.assertTrue(
-            "Can't parse patchwork url: '{}'".format(patchwork_url)
-            in context.exception
-        )
+    @mock.patch('requests.get', side_effect=mocked_requests_get)
```

Comment:
I don't know if you are thinking on writing more complicated test cases using `mocked_requests_get` but here with the following code will be enough:
```pythonI don't think it is worse, but I am not convinced it is better either hehe. I'm used to seeing comparisons using mock.call, maybe that's why it sounds more intuitive to me. Anyway I want to see this PR merged soon so I'll take the way you prefer and if in the future we find a better way to solve this, we can change it :):
    @mock.patch('requests.get', return_value=mock.Mock) 
    def test_get_patchwork_mbox(self, mock_get):
        mock_get().text = 'mbox text goes here'
```
I've tested it locally and it works ok
```diff
@@ -39,10 +39,11 @@ def mocked_requests_get(*args):
 class TestInit(unittest.TestCase):
     """Test cases for skt's __init__.py"""
 
-    @mock.patch('requests.get', side_effect=mocked_requests_get)
+    @mock.patch('requests.get', return_value=mock.Mock)
     def test_get_patchwork_mbox(self, mock_get):
-        # pylint: disable=unused-argument
         """Ensure get_patchwork_mbox() can get an mbox file"""
+        expected_text = 'mbox text goes here'
+        mock_get().text = expected_text
         patchwork_uri = 'https://patchwork.kernel.org/patch/10326957/'
         result = skt.get_patchwork_mbox(patchwork_uri)
-        self.assertEqual(result, 'mbox text goes here')
+        self.assertEqual(result, expected_text)
```